.github/workflows/android.yml

```diff
@@ -25,6 +25,10 @@ jobs:
       env:
          GOOGLE_SERVICE_JSON:  ${{ secrets.GOOGLE_SERVICE }}
       run:  echo $GOOGLE_SERVICE_JSON >  app/google-service.json
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew 
+    - name: Build with Gradle
+      run: ./gradlew build
```

Comment:
You are doing this right below, so I'm not sure what's the point with this pull request 🤔
It should surely not be merged as is.

Reply:
The pull request was just to test whether the google service needed to have its own independent build job because when we added the google service in the previous pull request, the app was still having the problem.